nservicebus/upgrades/support-policy.md

```diff
@@ -30,4 +30,13 @@ If a version is not listed or has expired but support is required [contact Parti
 
 ### NServiceBus 3.x
 
-As of 10 October 2015, Version 3 is no longer supported. If support is required [contact Particular](http://particular.net/contactus).
\ No newline at end of file
+As of 10 October 2015, Version 3 is no longer supported. If support is required [contact Particular](http://particular.net/contactus).
+
+
+## Compatibility guarantees
+
+The compatibility guarantees cover two aspects: upgrades and communication between endpoints in various versions (so called `wire compatibility`).
+
+The recommended approach is to upgrade 1 major version at a time, with deployment to production. That guarantees that the necessary transformations of data are performed and no information will be lost.
+
+The wire compatibility is guaranteed across 1 major version. Additionally, there's the best effort and extensive testing to cover compatibility across all versions, starting with NServiceBus Version 3.
```

Comment:
It feels a bit odd to lock the wire compat down to versions (since we want to be more aggressive with releasing major in the future if possible)

Should we consider using dates instead? or "we guarantee wire compatibility between all supported versions" ?